WebViewControl/WebViewLoader.cs

```diff
@@ -15,13 +15,14 @@ internal static class WebViewLoader {
         private static string[] CustomSchemes { get; } = new[] {
             ResourceUrl.LocalScheme,
             ResourceUrl.EmbeddedScheme,
-            ResourceUrl.CustomScheme
+            ResourceUrl.CustomScheme,
+            ResourceUrl.HttpsScheme
         };
 
         private static GlobalSettings globalSettings;
 
         [MethodImpl(MethodImplOptions.NoInlining)]
-        public static void Initialize(GlobalSettings settings) {
+        public static void Initialize(WebView ownerWebView, GlobalSettings settings) {
```

Comment:
cant do this ... this is called ran only once for the first webview... the other will return right away